highfidelity/hifi
# Fix 'Browse Avatars' button from Avatar Settings

Fixes [MS11361](https://highfidelity.manuscript.com/f/cases/11361/Unable-to-switch-avatar-from-marketplace-on-stable-and-on-dev-builds).

## Test Plan
Perform the steps below in Desktop mode, and then in HMD mode.
1. Go to Settings -> Avatar
2. Click "Browse Avatars"
3. Choose a new avatar to use, then verify that your avatar has changed.
